File: workspace/1_SumOfSquares/test/nums/SumOfSquaresTest.java
package nums;

import static junit.framework.Assert.assertEquals;

import org.junit.Test;
import java.util.ArrayList;
import java.util.Arrays;

public class SumOfSquaresTest {
	
	@Test
	public void emptyList() {
		assertEquals(new Integer(0), ListUtil.sumOfSquares(new ArrayList<Integer>()));
	}
	
	@Test
	public void sumOfSquares() {
		assertEquals(new Integer(55), ListUtil.sumOfSquares(Arrays.asList(1, 2, 3, 4, 5)));
	}

}

File: solution/1_SumOfSquares/1_SumOfSquares/SumOfSquaresTest.cs
using System;
using System.Text;
using System.Collections.Generic;
using System.Linq;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using nums;

namespace test
{
    [TestClass]
    public class SumOfSquaresTest
    {
        [TestMethod]
        public void emptyList() {
		    Assert.AreEqual(0, ListUtil.sumOfSquares(new List<int>()));
	    }

        [TestMethod]
	    public void sumOfSquares() {
            Assert.AreEqual(55, ListUtil.sumOfSquares(new List<int>(new int[] {1, 2, 3, 4, 5})));
	    }
    }
}
